keyboards/thedogkeyboard/config.h

```diff
@@ -0,0 +1,246 @@
+/*
+Copyright 2019 kakunpc
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
```

Comment:
```suggestion
#define PRODUCT_ID 0x000A
```

You may want to use a unique product ID. Currently, there's no actual benefit to it, especially since we don't have a fixed vendor ID. It just so happens that 0xF33D is unassigned.

Few things actually use this VID:PID pair, though I think VIA uses it to distinguish between boards.

I suggested 000A as it seems unused in QMK so far. It's just a small suggestion, and not a requirement :)